queries.sql

```diff
@@ -15,4 +15,53 @@ SELECT * FROM animals WHERE neutered = 'true';
 -- Find all animals not named Gabumon.
 SELECT * FROM animals WHERE name != 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg = 10.4 OR 17.3 OR weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg = 10.4 OR 17.3 OR weight_kg BETWEEN 10.4 AND 17.3;
+-- nside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
-  [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) for this project, " `Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction` ", please kindly verify that change was made, something like so: 

```sql
  UPDATE animals
   SET species = 'unspecified';

   SELECT species from animals; -- this is to verify that a change was made
   ROLLBACK;

   SELECT species from animals; -- this is to verify that a change was made
```